packages/synchro-charts/src/globals/awsui.css

```diff
@@ -16057,5 +16046,5 @@ body.awsui-modal-open {
 @media screen and (max-width: 992px) {
   .awsui .awsui-wizard__column-form .awsui-wizard__step-number {
     display: block;
-  }
```

Comment:
This unclosed block is causing a build failure. This closing brace needs to be added back in

Reply:
Yes, I fixed that on my local, there other changes that I did not pushed yet

Reply:
can you push them?